user/application/build.gradle

```
@@ -1,8 +1,10 @@
 dependencies {
     implementation project(':user:domain')
+    implementation project(':core:id-generator:id-core')
```

앗 이거 전달을 못드렸네요..!
id-generator 사용시에, 아래처럼 `id-generator-starter` 의존성 받으시면, core모듈이랑 구현체 모듈 자동으로 설정되게끔 만들어 놨습니다..!
아마 core만 받으시면 `tsid` 모듈이 안딸려올고 빈으로 등록되 안될거에요 ㅠㅠ

```suggestion
    implementation project(':core:id-generator:id-generator-starter')
```

---

b7571f4